setup.py

```diff
@@ -3,11 +3,15 @@
 
 setup(
     name="calc",
-    version="0.3",
+    version='0.4',
```

Comment:
Maybe save the `v1.0` bump until we flesh out the test suite and have full py3 compat?
Or just go for it and cut it now; this isn't exactly beta code anymore...

Reply:
Let's ship '0.4' as is, including the pin to pyparsing above.  We can have a follow up, to get py3 and better setup.py hygiene including unpinning in the install requirements.  That one will be a breaking change anyway because unpinning in the install_requires might cause side effectes upstream if they don't explicitly pin pyparsing correctly.